### PR TITLE
Fix remaining typos

### PR DESCRIPTION
**Pull request by luzpaz**
Folllow-up to #130

Note: does contain source changes (see below).

### CODE REVIEW

File: trunk/tools/ampsim/DK/gschem-schematics/build-plug.py [hunk 1]
```diff
@@ -70,12 +70,12 @@ def parse_vars(items):
...
-comandline += "**************************** File generated by *********************************\n"
-comandline += "********************************************************************************\n"
+commandline = "\n\n/*******************************************************************************\n"
+commandline += "**************************** File generated by *********************************\n"
+commandline += "********************************************************************************\n"
 for arg in sys.argv:
-    comandline += " %s" % arg
-comandline += "\n*******************************************************************************/\n\n"
+    commandline += " %s" % arg
+commandline += "\n*******************************************************************************/\n\n"
```
Comment (luzpaz) [inline trunk/tools/ampsim/DK/gschem-schematics/build-plug.py:78]: source change

File: trunk/tools/ampsim/DK/gschem-schematics/build-plug.py [hunk 2]
```diff
@@ -333,7 +333,7 @@ def write_final_file(self, a, dspfile,fdata,dspfileui,fuidata,freqs,gain_stages,
                 fdata += process_line
         if(faust_table):
             fdata = fdata.replace('import("stdfaust.lib");', 'import("stdfaust.lib");\n%s') % faust_table_template
-        fdata = comandline + fdata
+        fdata = commandline + fdata
```
Comment (luzpaz) [inline trunk/tools/ampsim/DK/gschem-schematics/build-plug.py:336]: source change

File: trunk/tools/pluginloader.pyx [hunk 3]
```diff
@@ -79,7 +79,7 @@ cdef extern from "gx_plugin.h":
         char* id
         char* name
         char** groups
-        char* decription
+        char* description
```
Comment (luzpaz) [inline trunk/tools/pluginloader.pyx:82]: source change

**Comment by brummer10** [comment]
Many thanks, @luzpaz

**Pull request merged**